src/main/scala/uk/gov/hmrc/play/http/HeaderNames.scala

```diff
@@ -20,20 +20,46 @@ object HeaderNames {
 
   import play.api.http.HeaderNames.AUTHORIZATION
 
-  val authorisation = AUTHORIZATION
-  val xForwardedFor = "x-forwarded-for"
-  val xRequestId = "X-Request-ID"
-  val xRequestTimestamp = "X-Request-Timestamp"
-  val xSessionId = "X-Session-ID"
-  val xRequestChain = "X-Request-Chain"
-  val trueClientIp = "True-Client-IP"
-  val trueClientPort = "True-Client-Port"
-  val token = "token"
-  val surrogate = "Surrogate"
-  val otacAuthorization = "Otac-Authorization"
-  val googleAnalyticTokenId = "ga-token"
-  val googleAnalyticUserId  = "ga-user-cookie-id"
-  val deviceID  = "deviceID" // not a typo, should be ID
+  /*
+   * this isn't ideal, but downstream apps still want to refer to typed header values
```

Comment:
I wonder if there's a better way of doing this...  maybe case objects or enumerations (sorry my scala isn't all that great)

Also is there a reason this is a map rather than a seq?